src/pmdas/denki/denki.c

```diff
@@ -693,6 +798,9 @@ main(int argc, char **argv)
         		strncpy(rootpath, opts.optarg, sizeof(rootpath));
 			rootpath[sizeof(rootpath)-1] = '\0';
             		break;
+		case 'C':
+                	pmSetDebug("appl0");
+                	break;
```

Comment:
Can drop these three lines.  The -D option is the standard everywhere in PCP for enabling debugging diagnostics, and this agent already supports it automatically via pmdaGetOptions.